<?xml version="1.0"?>
<launch>

  <!-- joy node -->
  <node name="joy_node" pkg="joy" type="joy_node" output="screen">
    <param name="autorepeat_rate"      type="double" value="20.0 "   />
  </node>

  <!-- joy2vel node -->
  <node name="francor_joy2vel_node" pkg="francor_joy2vel" type="francor_joy2vel_node" output="screen">
    <param name="joy_map"          type="string" value="ps4"   />
    <param name="rate"             type="double" value="50.0"   />
    <param name="max_lin_vel"      type="double" value="0.75 "   />
    <param name="max_ang_vel"      type="double" value="0.75"   />
    <!-- <param name="max_sh_vel"       type="double" value="35.0 "   /> -->
    <param name="dead_zone_sh"     type="double" value="0.2"   />
    <!-- <param name="pan_default"      type="int" value="0 "   /> -->
    <param name="tilt_default"     type="double" value="0.5"   />
    
    <remap from="/cmd_vel" to="/cmd_vel"/>

  </node>

<!-- francor victim -->
<!-- <include file="$(find francor_victim)/launch/default.launch"/> -->

<!-- hector geotiff -->
<!-- <include file="$(find hector_geotiff)/launch/geotiff_mapper_only.launch"/> -->

<!-- rqt_gui -->
<!-- <include file="$(find francor_rqt_plugins)/launch/default.launch"/> -->
  
  <!-- <include file="$(find francor_overlay_hack)/launch/default.launch"/> -->

  <!-- <node name="start_scd_sh" pkg="francor_joy2vel" type="start_scd.sh" output="screen"/> -->

</launch>
